modules/nf-core/plink/gwas/meta.yml

```diff
@@ -0,0 +1,79 @@
+name: "plink_gwas"
+description: Generate GWAS association studies
+keywords:
+  - association
+  - GWAS
+  - case/control
+tools:
+  - "plink":
+      description: "Whole genome association analysis toolset, designed to perform a range of basic, large-scale analyses in a computationally efficient manner."
+      homepage: "https://www.cog-genomics.org/plink"
+      documentation: "https://www.cog-genomics.org/plink/1.9/data#recode"
+      tool_dev_url: "https://www.cog-genomics.org/plink/1.9/dev"
+      licence: ["GPL"]
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - meta2:
+      type: map
+      description: |
+        Groovy Map containing sample information,
+        e.g. [ id:'test', single_end:false ]
+  - meta3:
+      type: map
+      description: |
+        Groovy Map containing sample information,
+        e.g. [ id:'test', single_end:false ]
+  - meta4:
+      type: map
+      description: |
+        Groovy Map containing sample information,
+        e.g. [ id:'test', single_end:false ]
+  - bed:
+      type: file
+      description: PLINK binary biallelic genotype table file
+      pattern: "*.{bed}"
+  - bim:
+      type: file
+      description: PLINK extended MAP file
+      pattern: "*.{bim}"
+  - fam:
+      type: file
+      description: PLINK sample information file
+      pattern: "*.{fam}"
+  - bcf:
+      type: file
+      description: PLINK variant information + sample ID + genotype call binary file
+      pattern: "*.{bcf}"
+  - vcf:
+      type: file
+      description: Variant calling file (vcf)
+      pattern: "*.{vcf}"
+  - phe:
+      type: file
+      description: PLINK file containing family IDs in the first column, within-family IDs in the second column and phenotype information in the third column
```

Comment:
this is not necessarily the file format though is it? one could specify the phenotype column name and have the phenotype in a column other than the third